DuggaSys/microservices/shared_microservices/getUid_ms.php

```diff
@@ -0,0 +1,28 @@
+<?php
+    date_default_timezone_set("Europe/Stockholm");
+    
+    // Include basic application services
+    include_once ("../../../../coursesyspw.php");
+    include_once ("../../../Shared/sessions.php");
+    include_once ("../../../Shared/basic.php");
+
+    function getUid(){
+        // Checks user id, if user has none a guest id is set
+        if(isset($_SESSION['uid'])){
+            $userid=$_SESSION['uid'];
+        }else{
+            $userid="1";
+        }
+
+        $log_uuid = getOP('log_uuid');
+        $log_timestamp = getOP('log_timestamp');
+
+        $log_uuid = getOP('log_uuid');
+        $info="opt: ".$opt." courseId: ".$courseId." courseVersion: ".$courseVersion." exampleName: ".$exampleName." sectionName: ".$sectionName." exampleId: ".$exampleId;
+        logServiceEvent($log_uuid, EventTypes::ServiceServerStart, "getUid_ms.php",$userid,$info);
```

Comment:
Not sure if this is working as intended, can't find any entries in `loglena6.db` with `getUid_ms.php`, I do however find my actions in `codeviewerService.php`

Reply:
That is copied from old code, didn't check if it actually works